### PR TITLE
Rebalance flat bonuses, add one refinement level to NAI and AA

### PR DESCRIPTION
**Pull request by Oberlus**
This is a follow up to #2943 

Gas Giant Generator down to +5 and +3 (from +10 and +5)

Nascent AI down to +1 from +2.
New tech refinement Artificial Minds +3.

Adaptive Automation down to +2 from +5.
New tech refinement Sentient Automation +6.

AI updated accordingly to get the new tech refinements.
I could not find any place in the AI code to tell it the new values for these techs.
@agrrr3  @Cjkjvfnby @Morlic-fo please review this changes.

[Related thread](https://www.freeorion.org/forum/viewtopic.php?f=6&t=11040&p=101798&sid=3f5e517fcafd674bbd677c1122909dc5#p101703)

### CODE REVIEW

**Comment by o01eg** [comment]
@Oberlus CI errors should be fixed.

**Comment by Oberlus** [comment]
@Vezzra 
I would add this to v0.4.10, to tackle the issue with flat bonuses being too strong (no-brainer) in the early game. This happened before #2943 and now is more serious (although pop-based bonuses are still the main source of income late game).
In my SP gameplay I found that getting AA and GGG (when enough GGs, which happens often) was a huge industry boost that left the AIs at 50% output from me when the effects kick in.

This PR also deals with the issue that NAI and AA are a must early game but have small effect late game, by adding two rather expensive refinements to each of those techs.

**Comment by Oberlus** [comment]
Fixed missing commas in TechsLists.py.
Added the two new files.

File: default/python/AI/AIDependencies.py [hunk 1]
```diff
@@ -277,10 +277,12 @@ class Tags:
 PRO_SOL_ORB_GEN = "PRO_SOL_ORB_GEN"
 PRO_MICROGRAV_MAN = "PRO_MICROGRAV_MAN"
 PRO_SINGULAR_GEN = "PRO_SINGULAR_GEN"
-PROD_AUTO_NAME = "PRO_SENTIENT_AUTOMATION"
+PROD_AUTO_1_NAME = "PRO_ADAPTIVE_AUTOMATION"
+#PROD_AUTO_2_NAME = "PRO_SENTIENT_AUTOMATION"
```
Comment (o01eg) [inline default/python/AI/AIDependencies.py]: ./AI/AIDependencies.py:281:1: E265 block comment should start with '# '

Comment (Oberlus) [reply default/python/AI/AIDependencies.py]: Fixed

File: default/python/AI/AIDependencies.py [hunk 2]
```diff
@@ -299,9 +301,9 @@ class Tags:
 
 EXOBOT_TECH_NAME = "PRO_EXOBOTS"
 
-SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
-SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
-SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
+# SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
+# SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
+# SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
```
Comment (agrrr3) [inline default/python/AI/AIDependencies.py:306]: why uncommenting arc disruptor techs?

Comment (Oberlus) [reply default/python/AI/AIDependencies.py:306]: I understand you mean "why commenting out".
They are not used anywhere in the code, AFAICT:

```
$ grep -r SHP_WEAPON_ARC_DISRUPTOR default/python/AI/*
default/python/AI/AIDependencies.py:SHP_WEAPON_ARC_DISRUPTOR_1 = "SHP_WEAPON_ARC_DISRUPTOR_1"
default/python/AI/AIDependencies.py:SHP_WEAPON_ARC_DISRUPTOR_2 = "SHP_WEAPON_ARC_DISRUPTOR_2"
default/python/AI/AIDependencies.py:SHP_WEAPON_ARC_DISRUPTOR_3 = "SHP_WEAPON_ARC_DISRUPTOR_3"
default/python/AI/AIDependencies.py:    "SR_ARC_DISRUPTOR": tuple(("SHP_WEAPON_ARC_DISRUPTOR_%d" % i, i) for i in [2, 3]),
default/python/AI/TechsListsAI.py:            "SHP_WEAPON_ARC_DISRUPTOR_1",
```

Code is only using the strings (within quotes), not the definitions (without the quotes).
See how the similar code in AIDependencies.py for SHP_WEAPON_1_1 doesn't require those definitions (do not exist):

```
default/python/AI/AIDependencies.py:    "SR_WEAPON_1_1": tuple(("SHP_WEAPON_1_%d" % i, 1) for i in [2, 3, 4]),
```

**Comment by agrrr3** [comment]
I will probably do a PR based on this one with some restructuring, please request some art for the technologies on the forum

**Comment by Oberlus** [comment]
> I will probably do a PR based on this one with some restructuring

Can I know the reason?
Should be more reasonable that you let me know what you don't like and I fix it? Thus I learn and am better prepared for future contributions.

**Comment by agrrr3** [comment]
> > I will probably do a PR based on this one with some restructuring
> 
> Can I know the reason?
> Should be more reasonable that you let me know what you don't like and I fix it? Thus I learn and am better prepared for future contributions.

Yes, that would be the gold standard. I am just too tired to walk you through today, changing is easier than explaining in some cases

also i find the topics in the change/check loop and not before. e.g. did you want  GRO_NANOTECH_MED as prereq? just found an error in the AI log that it had troubles to parse the NASCENT_AI.focs.txt file and the prereq are specified twice

**Comment by agrrr3** [comment]
Ok, I did not manage to make the pull request against your branch, so the code is there: #2965

Some bugsfixes, bugs found by starting a game with AIs and advancing for some turns and then grepping through the AI logs 
some refactoring for PRO_AUTO_* in AIDeps: PRO_AUTO_2 was not used in ColonisationAI before - added fixed industry bonus dict for that;   
techLists have a slot pattern, if e.g. you add two economic techs you have to use them (by adding two economic slots) in the ordered list. In this case i think the LRN_ARTIF_2 is the fifth economic slot and PRO_AUTO_2 is the seventh or eigth

You can add the commits to your commit if you like or we can propose to merge my PR. You decide.

**Comment by Vezzra** [comment]
> I would add this to v0.4.10, to tackle the issue with flat bonuses being too strong (no-brainer) in the early game.

Alright.

**Comment by Vezzra** [comment]
As far as I can see, this PR got superseded by #2965, and should be closed accordingly - @Oberlus, @agrrr3, can you confirm that?

**Pull request closed without merging**